transcription/email.go

```diff
@@ -0,0 +1,30 @@
+package transcription
+
+import (
+	"log"
+	"net/smtp"
+	"os"
+	"strings"
+)
+
+func sendEmail(from string, to []string, subject string, body string) {
+	// Set up authentication information.
+	auth := smtp.PlainAuth(
+		"",
+		os.Getenv("MAIL_EMAIL"),
+		os.Getenv("MAIL_PASSWORD"),
+		"smtp.gmail.com",
+	)
+
+	// Connect to the server, authenticate, set the sender and recipient,
+	// and send the email all in one step.
+	msg := []byte("To: " + strings.Join(to, ", ") + "\r\n" +
+		"Subject: " + subject + "\r\n" +
+		"\r\n" +
+		body + "\r\n")
+	err := smtp.SendMail("smtp.gmail.com:25", auth, from, to, msg)
+
+	if err != nil {
+		log.Fatal(err)
```

Comment:
We should talk at some point about how error handling is done in Go. The best thing to do here would be to return err so the **caller** has to deal with it.

One of my favorite things about Go is that errors are always handled. The key reason why is that error values are propagated up the call stack so the **right** code handles it. Imagine we call `sendEmail` from another function `foo`. If `sendEmail` has an error, then `foo` might know how to handle it. Or maybe `foo` just panics. Or maybe `foo` was called by `bar` so `foo` returns the error back to bar. Regardless, `sendEmail` should have **no say** in how this error is handled. It should just say "not my problem" and move on.